CalendarApp/GoogleCalendarService/BookingService.cs

```diff
@@ -4,23 +4,34 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using GoogleCalendarService.Manager;
 
 namespace GoogleCalendarService
 {
     public class BookingService
     {
-        private CalendarService _service;
-        private ILog _log;
+        private readonly CalendarService _service;
+        private readonly ILog _log;
+        public EventManager EventManager;
+        public string TimeZone;
```

Comment:
And what about public field? Probably better is to encapsulate it into property.

Anyway, can we suggest something instead of shared and passed field to determine time zone?